ui/apps/platform/src/Containers/VulnMgmt/Dashboard/VulnMgmtDashboardPage.js

```diff
@@ -92,6 +96,7 @@ const VulnDashboardPage = ({ history }) => {
     return (
         <>
             <DashboardLayout
+                banner={hasReadAccessForIntegration && <ScannerV4IntegrationBanner />}
```

Comment:
The structure of the VM Dashboard components required a little bit of rework in order to thread this banner into the place we need it in the DOM.